messages/convert.source-behavior.md

```diff
@@ -0,0 +1,70 @@
+# summary
+
+Enable a behavior of your project source files, and then update your Salesforce DX project to implement the behavior.
+
+# description
+
+Specifically, this command updates the "sourceBehaviorOption" option in the "sfdx-project.json" file and then converts the associated local source files in your project as needed.
+
+For example, run this command with the "--behavior decomposePermissionSet" flag to start decomposing permission sets when you deploy or retrieve them. Decomposing means breaking up the monolithic metadata API format XML file that corresponds to a metadata component into smaller XML files and directories based on its subtypes. Permission sets are not decomposed by default; you must opt-in to start decomposing them by using this command. When the command finishes, your "sfdx-project.json" file is updated to always decompose permission sets, and the existing permission set files in your local package directories are converted into the new decomposed format. You run this command only once for a given behavior change.
+
+For more information about the possible values for the --behavior flag, see the "sourceBehaviorOptions" section in the https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_ws_config.htm topic.
```

Comment:
Just confirming this is going to be updated (cc @jshackell-sfdc)

Reply:
My thought was to NOT include the `Beta` label in the `--help` because then we have to remember to update it for GA. But if you think it's better to show it now, then that's fine.  I realize that the example wouldn't work in its original form until GA...

Reply:
Well, the command does tell you what the correct behavior names are in the error message:
```
> sf project convert source-behavior --behavior decomposePermissionSet
Warning: This command is currently in beta. Any aspect of this command can change without advanced notice. Don't use beta commands in your scripts.
Error (2): Expected --behavior=decomposePermissionSet to be one of: decomposeCustomLabelsBeta, decomposePermissionSetBeta, decomposeSharingRulesBeta, decomposeWorkflowBeta
See more help with --help
```

...but the `See more help with --help` would be confusing if we don't update them to the correct names (with Beta)

Reply:
Oh sorry, and by "this", I meant the https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_ws_config.htm docs page. It does not currently have a `sourceBehaviorOptions` section

Reply:
I checked in an update with that new section last week, it _should_ show up tomorrow.  Cross your fingers!